src/StrictMocking.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lendable\PHPUnitExtensions;
+
+use PHPUnit\Event;
+use PHPUnit\Framework\MockObject\Generator\Generator as MockObjectGenerator;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @mixin TestCase
+ */
+trait StrictMocking
+{
+    protected function createMock(string $originalClassName): MockObject
```

Comment:
```suggestion
    final protected function createMock(string $originalClassName): MockObject
```

Doesn't change much but intent.